index.js

```diff
@@ -339,13 +340,13 @@ function session(options) {
         return writetop();
       } else if (storeImplementsTouch && shouldTouch(req)) {
         // store implements touch method
-        debug('touching');
+        debug('touching the session')
```

Comment:
Any reason this ones contains `"the"` while the others (like `destroying session`, `generating session`, etc.) does not? Should all the others contain "the" or should this one remove the "the"?

Reply:
fixed it, removed `the`